Sources/Snowplow/Events/MessageNotification.swift

```diff
@@ -196,4 +196,130 @@ public class MessageNotification : SelfDescribingAbstract {
         payload[kSPMessageNotificationParamTrigger] = triggerToString(trigger)
         return payload
     }
+    
+    // MARK: - Builders
+    
+    /// The action associated with the notification.
+    @objc
+    public func action(_ action: String?) -> Self {
+        self.action = action
+        return self
+    }
+    
+    /// Attachments added to the notification (they can be part of the data object).
+    @objc
+    public func attachments(_ attachments: [MessageNotificationAttachment]?) -> Self {
+        self.attachments = attachments
+        return self
+    }
+    
+    /// The notification's body.
+    @objc
+    public func body(_ body: String) -> Self {
+        self.body = body
+        return self
+    }
+    
+    /// Variable string values to be used in place of the format specifiers in bodyLocArgs to use to localize the body text to the user's current localization.
+    @objc
+    public func bodyLocArgs(_ args: [String]?) -> Self {
+        self.bodyLocArgs = args
+        return self
+    }
+    
+    /// The key to the body string in the app's string resources to use to localize the body text to the user's current localization.
+    @objc
+    public func bodyLocKey(_ key: String?) -> Self {
+        self.bodyLocKey = key
+        return self
+    }
+    
+    /// The category associated to the notification.
+    @objc
+    public func category(_ category: String?) -> Self {
+        self.category = category
+        return self
+    }
+    
+    /// The application is notified of the delivery of the notification if it's in the foreground or background, the app will be woken up (iOS only).
+    public func contentAvailable(_ available: Bool?) -> Self {
+        self.contentAvailable = available
+        return self
+    }
+    
+    /// The group which this notification is part of.
+    @objc
+    public func group(_ group: String?) -> Self {
+        self.group = group
+        return self
+    }
+    
+    /// The icon associated to the notification (Android only).
```

Comment:
there are some functions here labelled as Android-only, is it true? should they be here?

Reply:
Yeah, they are Android-only in the sense that iOS doesn't automatically assign them, but I think there is no harm letting users use them if they want to.